ci/Jenkinsfile.linux

```diff
@@ -39,7 +39,7 @@ pipeline {
     booleanParam(
       name: 'USE_MOCKED_KEYCARD_LIB',
       description: 'Decides whether the mocked status-keycard-go library is built.',
-      defaultValue: false
+      defaultValue: true
```

Comment:
hmm @anastasiyaig to make this dynamic i.e setting this to `true` only for nightly would require us to figure out if the current job is a PR job or a nightly job or a release job.

Now I do see a utility in `status-jenkins-lib` that provides a boolean for `isPRbuild` 
https://github.com/status-im/status-desktop/blob/248ba1c1c8136c9e1bf6dd79e069a1336711832a/ci/Jenkinsfile.linux#L5
But I would assume that would not be enough.

So we might need a new utility for this which lives in `status-jenkins-lib` that tells is if this is a nightly job or not and we can then make `USE_MOCKED_KEYCARD_LIB ` dependant on that.

Reply:
You could just use the `?:` form like some options above do to preserve last used value, instead of trying to detect type of build.

But if you want to do that you can just use `isNightlyBuild()` from `utils`. But that's a more implicit behavior.

Reply:
ah thanks for explaining @jakubgs I do like `utils.isNightlyBuild()`